pandas/tests/series/test_constructors.py

```diff
@@ -2007,6 +2008,50 @@ def test_series_constructor_ea_int_from_string_bool(self):
         with pytest.raises(ValueError, match="invalid literal"):
             Series(["True", "False", "True", pd.NA], dtype="Int64")
 
+    @pytest.mark.parametrize("val", [1, 1.0])
+    def test_series_constructor_overflow_uint_ea(self, val):
```

Comment:
do these tests go through meaningfully different paths? if not id suggest combining them.  not a big deal

Reply:
Expected is annoying to define in one df, so would prefer keeping it separate to make it easier to read